wiki/News_styling_criteria/en.md

```diff
@@ -68,61 +68,34 @@ date: 2017-08-17 03:00:00 +0000
 
 ### Markdown
 
 The osu! wiki uses a partial implementation of GFM (GitHub Flavored Markdown) with the following exceptions:
 
-#### Titles
-
-News articles do not use traditional Markdown titles in any capacity whatsoever.
-
-#### Headers
-
-Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. They have unique identifiers and can be linked directly to. Never use headers to style or format text.
-
-#### Bold
-
-Bolding is reserved for highlighting objects or phrases of importance, be them players, scoring, or whatever else is contextually considered a focus of the article in question.
-
-#### Italics
-
-Italics is used uncommonly to accent phrases or statements of interest within the conversational register. It may also be used to designate names of other creative works, such as the *osu! Coffee Hour* or the *Aspire Beatmapping Contests*.
-
-#### Horizontal bar
-
-Horizontal bars must be used sparingly and only in situations where content needs to be cleanly and definitively separated from the rest of the article.
-
-#### Tables
-
-Markdown tables can be used, but is not recommended for displaying data in news articles. Visualising the data using an image is likely to be more pleasing and more accessible to most users.
-
-### Images
-
-All images linked to news articles must either be hosted on osu! servers (i.e. `assets.ppy.sh`) or placed within and referenced from the `osu-wiki` shared image repository.
-
-News articles that use images have their own folder within ``/wiki/shared/news/`` using the same naming format as their filename, with the exception of the ``.md`` suffix.
-
-Example:
-
-``/wiki/shared/news/2017-08-17-news-article-standards/banner.jpg``
-
-#### Covers
+- Titles
+- Headers
+  - Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. They have unique identifiers and can be linked directly to. Never use headers to style or format text.
+- Bold
+- Italics
+- Horizontal bar
+- Tables
+- Images
+  - Images linked to news articles must either be hosted on osu! servers (i.e. `assets.ppy.sh`) or placed within and referenced from the `osu-wiki` shared image repository. News articles that use images have their own folder within ``/wiki/shared/news/`` using the same naming format as their filename, except for the ``.md`` suffix. Example:
+  ``/wiki/shared/news/2017-08-17-news-article-standards/banner.jpg``
+- Covers
+  - Cover images should have a minimum width of 900 and minimum height of 200 for their base size. A `@2x` version with each dimension doubled should be provided if the source image is large enough to support it. Cover images appearing in multiple news articles are commonly put in the `/wiki/shared/news/banners` directory. <!-- TODO: there should be a more general place to explain how and why @2x images can be used, this isn't enough info and it's not only for cover images. -->
```

Comment:
this list barely contains any information since the explanations are removed

where appropriate, link terms to asc. most of these should likely be removed if there's nothing specific to news posts to say about them

Reply:
I was thinking to remove all of these as I do not really understand the need for most of them, as for the link I can do that but just wanted to confirm that this is something that's appropriate to have on the wiki

Reply:
removed the unnecessary stuff and pointed to ASC in sentence above, 545f0d9198fbeaae803d1214ffa2ef61cbc26ea1